### HYPOTHESIS TESTING

##### Background:
- Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.


##### Objective:
- To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.


##### Data Provided:
- The theoretical weekly operating cost model: W = $1,000 + $5X
- Sample of 25 restaurants with a mean weekly cost of Rs. 3,050
- Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units

In [1]:
import math 
from scipy.stats import norm

In [2]:
sample_mean = 3050
x = 600
sample_size = 25

In [3]:
mu = 1000 + 5*x

In [4]:
sigma = 5*25

In [5]:
z = (sample_mean - mu) / (sigma / math.sqrt(sample_size))
print("Theoretical Mean:", mu)
print("Population Standard Deviation:", sigma)
print("Z Statistic:", z)

Theoretical Mean: 4000
Population Standard Deviation: 125
Z Statistic: -38.0


In [6]:
alpha = 0.05
critical_value = norm.ppf(0.95)
print("Critical Value:", critical_value)

if z > critical_value:
    print("Reject the Null Hypothesis")
else:
    print("Fail to Reject the Null Hypothesis")

Critical Value: 1.6448536269514722
Fail to Reject the Null Hypothesis


##### Conlusion
- At the 5% significance level, the calculated z-value (-38) is less than the critical value (1.645). Therefore, we fail to reject the null hypothesis. There is insufficient statistical evidence to support the restaurant owners' claim that the weekly operating costs are higher than those predicted by the theoretical cost model.